test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1156,9 +1154,8 @@ def run_copy(self, hosts=None, src=None, dest=None, sudo=False, uid=None,
                 cmd += copy_cmd
                 if recursive:
                     cmd += ['-r']
-                if (self.get_platform() == 'shasta' and _runas_user and
-                   _runas_user.port):
-                    cmd += ['-P', _runas_user.port]
+                if self.get_platform() == 'shasta' and runas and runas.port:
```

Comment:
Again, I feel that the shasta check is not needed here, if runas and runas.port is set, I think we should just use it.